src/util/console-color.ts

```diff
@@ -281,4 +287,13 @@ export const coloredConsoleStyles = `
   .log-bg-white {
     background-color: rgb(255, 255, 255);
   }
+  @media not screen {
```

Comment:
Is this for when the page is printed?

Reply:
yes, just incase someone uses the print this will to show `[redacted]` instead of the secret...

Reply:
ok